README.md

```diff
@@ -86,7 +85,11 @@ Run the quickstart app on an Android device
 <img height="500px" src="images/quickstart/voice_activity.png">"
 
 
-Press the call button to connect to Twilio
+Press the call button to connect to Twilio.
+
+<img height="500px" src="images/quickstart/voice_make_call_dialog.png">
+
+Press the call button to start a call. You will hear the congratulatory message.
```

Comment:
Leave the dialog text field empty and press the call button to start a call. You will hear the congratulatory message. Dialing a number or another client is described in steps 10 and 11.